cli/pcluster/utils.py

```diff
@@ -235,18 +231,34 @@ def get_stack_output_value(stack_outputs, output_key):
     return next((o.get("OutputValue") for o in stack_outputs if o.get("OutputKey") == output_key), None)
 
 
-def verify_stack_creation(cfn_client, stack_name):
+def get_stack(stack_name, cfn_client=None):
+    """
+    Get the output for a DescribeStacks action for the given Stack.
+
+    :param stack_name: the CFN Stack name
+    :param cfn_client: boto3 cloudformation client
+    :return: the Stack data type
+    """
+    try:
+        if not cfn_client:
+            cfn_client = boto3.client("cloudformation")
+        return cfn_client.describe_stacks(StackName=stack_name).get("Stacks")[0]
+    except ClientError as e:
```

Comment:
What if the stack doesn't exist? Will this throw a `ClientError` or an `IndexError` which we don't handle.

Reply:
If the stack doesn't exist the describe_stacks raises a `ClientError` so we cannot incur in the `IndexError`, in any case I added the handle for the IndexError too.

```
$ pcluster instances test
ERROR: Stack with id parallelcluster-test does not exist
```